packages/components/tour/src/mask.vue

```diff
@@ -3,14 +3,14 @@
     v-if="visible"
     :class="ns.e('mask')"
     :style="({
-      position: 'fixed',
-      left: 0,
-      right: 0,
-      top: 0,
-      bottom: 0,
-      zIndex,
-      pointerEvents: pos && targetAreaClickable ? 'none' : 'auto',
-    } as any)"
+    position: 'fixed',
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    zIndex,
+    pointerEvents: pos && targetAreaClickable ? 'none' : 'auto',
+  } as any)"
```

Comment:
There seems to be a problem with the code formatting.

Reply:
This is what happens after I execute `pnpm format`.

Reply:
OK